### PR TITLE
chore: add CODEOWNERS

### PR DESCRIPTION
**Pull request by rosvik**
Based on https://github.com/AtB-AS/docs-private/blob/main/kundevendt-development-guidelines.md

But took the liberty to add this part:

```
# Native code
/android/ @gorandalum @reidzeibel @jorelosorio
/ios/ @gorandalum @jorelosorio @reidzeibel
```

If that's ok with our native experts 😊 @reidzeibel @jorelosorio

### CODE REVIEW

**Review by reidzeibel — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Mobile token (App, Webshop, Token-server)
+/src/mobile-token/ @gorandalum @jorelosorio @hanne-at-atb
+
+# Mobility (App, BFF)
+/src/mobility/ @gorandalum @marius-at-atb @reidzeibel
+
+# Builds (GH Actions, App Store, Play Store, App Center)
+/.github/workflows/ @gorandalum @jorelosorio @reidzeibel
+/env/ @gorandalum @jorelosorio @reidzeibel
+/fastnane/ @gorandalum @jorelosorio @reidzeibel
```
Comment (reidzeibel) [inline .github/CODEOWNERS:16]:
```suggestion
/fastlane/ @gorandalum @jorelosorio @reidzeibel
```

Comment (reidzeibel) [reply .github/CODEOWNERS:16]: a small typo I think

**Review by reidzeibel — CHANGES_REQUESTED** [review]

**Review by marius-at-atb — APPROVED** [review]
Nice!
See suggestion by @reidzeibel before merging

File: .github/CODEOWNERS (hunk 1 above)
Comment (rosvik) [reply .github/CODEOWNERS:16]: 🙈

**Review by reidzeibel — APPROVED** [review]

**Pull request merged**